pkg/cmd/inspect.go

```diff
@@ -155,24 +202,99 @@ func generateCatalog() (*camel.RuntimeCatalog, error) {
 		Provider: v1.RuntimeProviderMain,
```

Comment:
`v1.RuntimeProviderQuarkus` is the default and the only one shortly.

Reply:
Just changed it.